src/editor/Core/Theme.re

```diff
@@ -34,9 +35,7 @@ let default: t = {
   editorMenuBackground: Color.hex("#2F3440"),
   editorMenuForeground: Color.hex("#FFFFFF"),
   editorMenuItemSelected: Color.hex("#495162"),
+  tabModifiedIconColor: Color.hex("#737984"),
```

Comment:
I've been trying to stick with the VSCode theme color nomenclature -  documented here: https://code.visualstudio.com/api/references/theme-color 

Just to make it easy to synchronize once we support VSCode themes. Do you know what the equivalent VSCode theme color name is? I looked through the tab ones like `tab.activeForeground` and wasn't sure which one is used for this dirty icon (it seems like the same color is used for the `x` and the dirty icon) 🤔

Reply:
@bryphe sorry spent a lot more time in their repository than in the editor I don't know much about that, it looks like its a similar color to the text I can just create a key for the color of the text and use that to style the icon

Reply:
That sounds reasonable. I'm trying to get our themes to match up as much as I can to their theme colors so that its easy to map once we start loading their themes - will save me some time once we hook up VSCode themes!